lib/plain/common-types.ts

```diff
@@ -857,4 +857,6 @@ export type AlphaExtensions = {
   }
 }
 
+export type AlphaExtensions = AlphaWorkflowExtension
```

Comment:
what is the benefit of doing so vs leaving it the way it was? I don't see the difference, but now we have 2 types which mean the same thing but named differently

Reply:
I guess my goal was to start with a more general extensions type that would later be a union of all alpha features. i'll remove it for now

Reply:
![Screenshot 2021-12-15 at 15 57 18](https://user-images.githubusercontent.com/9327071/146209721-b20d8d8e-c401-4795-b5c4-203f9b2cd685.png)

I would like to keep it for this helper function. Makes more sense to have a general union type for all alpha features to come